app/app.py

```diff
@@ -186,3 +232,10 @@ def get_trip_features(intersect_ids, grid, flow, mode):
 @app.exception(exceptions.NotFound)
 async def ignore_404s(request, exception):
     return response.text("Page not found: {}".format(request.url))
+
+#
+# TODO: does this break the app deployment? Handy for local deve but seem to remember
+# TODO: a good reason for removing it
```

Comment:
Seems like this could be bad. Are we just going to merge and hope the deployment doesn't break/rollback if necessary? Thought Travis might allow us to preview Pull Requests but I didn't see this PR in there.

Reply:
good catch. i meant to leave `app.run()` commented out. i've made that change.